Jenkinsfile

```diff
@@ -123,7 +124,7 @@ pipeline {
 
                             //stash(allowEmpty: true, includes: 'build/compile_commands.json', name: 'Compile commands')
                             //stash(allowEmpty: true, includes: 'build/reports/', name: 'Reports')
-                            archive(includes: 'build/bin/,compile_commands.json')
+                            //archive(includes: 'build/bin/,compile_commands.json')
```

Comment:
Why do we need this commented lines?

Reply:
it is temporary. it saves files in host but does not use.